packages/ffe-form-react/src/PhoneNumberField.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import {string, bool, func} from 'prop-types';
+
+import i18n from './i18n/i18n';
+
+export default function PhoneNumberField({
+                               phone,
+                               countryCode,
+                               onChange,
+                               onBlur,
+                               locale,
+                               disabled,
+                               showValidationError,
+                               className
+                           }) {
+
+    const supportedLocales = ['nb', 'nn', 'en'];
+    const text = i18n[supportedLocales.indexOf(locale) !== -1 ? locale : 'nb'];
+
+    function onPhoneChange(e) {
+        onChange(countryCode, e.target.value);
+    }
+
+    function onCountryCodeChange(e) {
+        onChange(e.target.value, phone);
+    }
+
+    function handleBlur(e) {
+        if (e.target.id !== 'country-code' || e.target.value.length === 0) {
+            onBlur(e);
+        }
+    }
+
+    return (
+        <div className={`ffe-input-group ${className ? className : 'phone-number-field'}`}>
```

Comment:
Looks like the `phone-number-field` class is applied to two `div`s here - this one and its direct child